### PR TITLE
Feature/Ecom-38-address-details

### PR DESCRIPTION
**Pull request by YasmeenGad**
+ 
WhatsApp Image 2024-12-13 at 12 48 55 AM

+ 
WhatsApp Image 2024-12-13 at 12 48 55 AM (1)

+ 
WhatsApp Image 2024-12-13 at 12 48 56 AM

+ 
WhatsApp Image 2024-12-13 at 12 48 56 AM (1)

### CODE REVIEW

**Review by haidyhesham95 — APPROVED** [review]

File: lib/features/address/presentation/view/address_screen.dart [hunk 1]
```diff
@@ -14,11 +23,60 @@ class AddressScreen extends StatefulWidget {
...
+  }
+
+  @override
+  void dispose() {
+    addressController.dispose();
+    phoneController.dispose();
+    recipientController.dispose();
+    super.dispose();
+  }
+
```
Comment (Marco-Nagy) [inline lib/features/address/presentation/view/address_screen.dart:53]: محتاجين نهندل الكلام ده فى الview model احسن

Comment (Marco-Nagy) [reply lib/features/address/presentation/view/address_screen.dart:53]: بحيث الfunctions كلها تبقى فى الview Model

**Review by Marco-Nagy — CHANGES_REQUESTED** [review]
please build your functions at view model class and call them as an actions

**Review by somaya-mahmoud — APPROVED** [review]

File: lib/features/address/presentation/view/address_screen.dart (hunk 1 above)
Comment (YasmeenGad) [reply lib/features/address/presentation/view/address_screen.dart:53]: Done

**Review by Marco-Nagy — APPROVED** [review]

**Review by somaya-mahmoud — APPROVED** [review]

**Pull request closed without merging**